src/App.js

```diff
@@ -1,21 +1,38 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import ProductDetail from './components/ProductDetail';
- import footer from './components/footer.js';
-  import Header from './components/Header.js';
- import Carousel from './components/Carousel.js';
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import ProductDetail from "./components/ProductDetail";
+import footer from "./components/Footer.js";
+import Header from "./components/Header.js";
+import Carousel from "./components/Carousel.js";
+import state from "./state";
 
 
 function App() {
-    return (
-      <div>
+  const products = state.products;
+  console.log(products);
+  return (
+    <div>
+      {products.map(product => {
```

Comment:
Good job mapping here, but you don't need to send everything separately. You could just send the entire product product={product}